src/core/default/commands/relationaldb/shell.py

```diff
@@ -23,16 +23,52 @@ def add_arguments(self, parser: ArgumentParser) -> None:
             type=str,
             help="The database to execute on. Name must include component name. ex: comp1.myDb",
         )
+        parser.add_argument(
+            "--c",
```

Comment:
This should be `-c` and then `--command=command` for the long form paramter
-c command
--command=command

see options sections of https://www.postgresql.org/docs/current/app-psql.html#:~:text=psql%20can%20be%20told%20about,database%20name%20is%20already%20given).